tests/__init__.py

```diff
@@ -1,5 +1,29 @@
 import datetime
+import time
+import functools
 
 
 def get_version():
     return datetime.datetime.utcnow().strftime("%Y-%m-%dT%H%M%S.%fZ")
+
+
+def eventually(timeout: float, interval: float):
+    """
+    eventually is designed to run a test multiple times until a the assertion is satisfied or a timeout is reached.
```

Comment:
s/a the/the

Reply:
Or actually `until all assertions are satisfied`

Reply:
Good catch. Made this change too.